app/controllers/logs_controller.rb

```diff
@@ -0,0 +1,38 @@
+class LogsController < ApplicationController
+
+  # A table of all the call logs in the system.
+  def index
+    @logs = Log.all.order("created_at DESC")
```

Comment:
Make sure to add that index so we can order using an index.

Also, don't load all your data at once.